colorbleed/plugins/maya/publish/collect_animation.py

```diff
@@ -0,0 +1,63 @@
+import pyblish.api
+
+import maya.cmds as cmds
+
+
+class CollectLook(pyblish.api.InstancePlugin):
```

Comment:
Voila, this should be `CollectAnimation`, or `CollectionAnimationData`, or whatever. :)

Also update the docstring. :)